docs/changelog/1315.md

```diff
@@ -0,0 +1 @@
+- Added support to export gradient data in VTK exporting
```

Comment:
The changelog is for user and developer. Could you also mention how the gradient data is named in the vtk files. @uekerman do you agree on the naming convention? We have now `<ScalarGradient>_gradient` and `<VectorData>_dx`,`<VectorData>_dy`,`<VectorData>_dz`.

Reply:
OK for me, there is probably no perfect solution. This one is good.